core/example/CMakeLists.txt

```diff
@@ -1,4 +1,3 @@
-option(Cabana_ENABLE_EXAMPLES "Build examples" OFF)
```

Comment:
Why?

Reply:
Because I'm a bad person. I think this got accidently smushed in the merge. reverted